From 8c9778751d7dd7aacd1e49ccf375b2beef5aeb97 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.jz@renesas.com>
Date: Thu, 5 Nov 2020 17:04:35 +0700
Subject: [PATCH 449/452] arm64: dts: renesas: cat875: enable canfd channel 0

CANFD channel 0 use same pin mux with CAN0, therefore only
one can be enabled at once.

This commit enable CANFD channel 0, thus have to disable CAN0.

Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/cat875.dtsi | 16 +++++++++++-----
 1 file changed, 11 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/cat875.dtsi b/arch/arm64/boot/dts/renesas/cat875.dtsi
index aaefc3a..865f7fa 100644
--- a/arch/arm64/boot/dts/renesas/cat875.dtsi
+++ b/arch/arm64/boot/dts/renesas/cat875.dtsi
@@ -30,10 +30,14 @@
 	};
 };
 
-&can0 {
-	pinctrl-0 = <&can0_pins>;
+&canfd {
+	pinctrl-0 = <&canfd_pins>;
 	pinctrl-names = "default";
 	status = "okay";
+
+	channel0 {
+		status = "okay";
+	};
 };
 
 &can1 {
@@ -54,9 +58,11 @@
 		};
 	};
 
-	can0_pins: can0 {
-		groups = "can0_data";
-		function = "can0";
+	canfd_pins: canfd {
+		channel0 {
+			groups = "canfd0_data";
+			function = "canfd0";
+		};
 	};
 
 	can1_pins: can1 {
-- 
2.7.4

